doc/release-notes/release-notes-pr12255.md

```diff
@@ -0,0 +1,17 @@
+PR #12255
```

Comment:
Thanks for adding a release notes item.
I'd suggest `systemd init file` as title, the PR number won't tell most users much